### PR TITLE
Blog post about concurency models in JetStream Super-Clusters

### PR DESCRIPTION
**Pull request by jnmoyne**
Blog post about the different concurency models in JetStream Super-Clusters and how you can have both immediate and eventual consistency at the same time and how to create a virtual globally replicated eventually consistent stream.

### CODE REVIEW

File: content/blog/multi-cluster-consistency-models.md [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+After a few seconds you can see that all the messages where are now present in all the 'read' streams, though not necessarily in the same order by comparing the output of
+```bash
+nats --context sc-west stream view foo
+```
+With
+```bash
+nats --context sc-central stream view foo
+```
+
+# What if you still want global ordering?
```
Comment (gcolliso) [inline content/blog/multi-cluster-consistency-models.md:375]: There is some controversy with using the word 'master' - we may want to switch to 'main' or 'primary'.

Comment (gcolliso) [reply content/blog/multi-cluster-consistency-models.md:375]: @jnmoyne Could you add the 'About the Author' section at the end, please?

Comment (wallyqs) [reply content/blog/multi-cluster-consistency-models.md:375]: yeah maybe reword this section to highlight that you have read streams mirrors that are following the original stream

File: content/blog/multi-cluster-consistency-models.md [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+- Stream consumers are 'per region', meaning you do not have a global named durable consumer on the virtual stream, but multiple regional ones.
+- Deleting individual messages from a virtual stream is not possible, the delete operation will only apply to the local 'read' stream and are not propagated (and neither should they be, as the message sequence numbers are not homogeneous between regions).
+- Compare-and-set operations are not possible on the virtual stream as you would be reading from one stream and writing to another and the message sequence numbers are not preserved between them.
+
+# Walkthrough
+In this example, we're going to walk through setting up a local Super-Cluster and creating a virtual stream 'foo' 
+Make sure to install (or upgrade to) the latest version of the [NATS server](https://github.com/nats-io/nats-server/releases/latest) and of the [`nats` CLI tool](https://github.com/nats-io/natscli#installation) on your local machine.
+
+```bash
+git clone https://github.com/ConnectEverything/eventually_consistent_global_stream.git
```
Comment (wallyqs) [inline content/blog/multi-cluster-consistency-models.md:116]: URLs are more natural with hyphens, maybe switch to that instead for the example repo?
```suggestion
git clone https://github.com/ConnectEverything/eventually-consistent-global-stream.git
```

File: content/blog/multi-cluster-consistency-models.md (hunk 1 above)
Comment (jnmoyne) [reply content/blog/multi-cluster-consistency-models.md:381]: Changed

File: content/blog/multi-cluster-consistency-models.md (hunk 2 above)
Comment (jnmoyne) [reply content/blog/multi-cluster-consistency-models.md]: Done. Moved the files to a new public repo in the Synadia labs org where it belongs more
